frontend/src/Component/Chat/index.tsx

```diff
@@ -1,14 +1,65 @@
 import { Box } from '@mui/material';
 import ChatList from './ChatList';
 import ChatDetail from './ChatDetail';
-import { useState } from 'react';
+import { useState, useLayoutEffect } from 'react';
+import { ChatType } from '../../Util/Type';
+import { io } from 'socket.io-client';
```

Comment:
import순서도 라이브러리에 가져온 것 따로 모으고, 개발자가 만든 파일에서 가져온거 따로 모으면 좋을 것 같아요! 

```suggestion
import { Box } from '@mui/material';
import { useState, useLayoutEffect } from 'react';
import { io } from 'socket.io-client';

import ChatList from './ChatList';
import ChatDetail from './ChatDetail';
import { ChatType } from '../../Util/Type';
```